multicluster/controllers/multicluster/common/controller_utils.go

```diff
@@ -78,3 +81,49 @@ func parseServiceCIDRFromError(msg string) (string, error) {
 func NewClusterInfoResourceExportName(clusterID string) string {
 	return clusterID + "-clusterinfo"
 }
+
+func ValidateLocalClusterClaim(c client.Client, clusterSet *mcv1alpha2.ClusterSet) (clusterID ClusterID, err error) {
+	configNamespace := clusterSet.GetNamespace()
+
+	clusterClaimList := &mcv1alpha2.ClusterClaimList{}
+	if err = c.List(context.TODO(), clusterClaimList, client.InNamespace(configNamespace)); err != nil {
+		return
+	}
+	if len(clusterClaimList.Items) == 0 {
+		err = fmt.Errorf("ClusterClaim is not configured for the cluster")
+		return
+	}
+
+	wellKnownClusterClaimIDExist := false
+	for _, clusterClaim := range clusterClaimList.Items {
+		if clusterClaim.Name == mcv1alpha2.WellKnownClusterClaimID {
+			wellKnownClusterClaimIDExist = true
+			clusterID = ClusterID(clusterClaim.Value)
+			break
```

Comment:
Why not return here? Then we can also remove `wellKnownClusterClaimIDExist`

Reply:
Done, thanks.